### PR TITLE
Benchmarks

### PR DESCRIPTION
**Pull request by marc-mabe**
... like the one in zend-eventmanager

I only created benchḿaks for the `Memory` and for the `Filesystem` storage adapters as these are the only one working from scratch.

### CODE REVIEW

**Comment by marc-mabe** [comment]
@Ocramius Can we merge this? I know it's not a full suite but it's a start.

**Comment by Ocramius** [comment]
@marc-mabe don't have time to manage merges atm, sorry :-(

File: composer.json [hunk 1]
```diff
@@ -22,7 +22,8 @@
         "zendframework/zend-serializer": "^2.6",
         "zendframework/zend-session": "^2.5",
         "fabpot/php-cs-fixer": "1.7.*",
-        "phpunit/PHPUnit": "~4.0"
+        "phpunit/PHPUnit": "~4.0",
+        "athletic/athletic": "dev-master"
```
Comment (Ocramius) [inline composer.json]: note: you may want to aim at phpbench (much better maintenance atm). See https://github.com/zendframework/zend-servicemanager/blob/3c3c3e5d090ae6adb945565529a713bdffd8b330/composer.json#L24

Comment (Ocramius) [inline composer.json]: The output standard deviation corrections are worth it: instant benefit and no more re-running the benchmarks a dozen times ;-)

**Comment by marc-mabe** [comment]
@Ocramius I have refactored it to use PHPBench now

**Pull request closed without merging**